packages/web/src/client/components/Grids/GridTeam.stories.tsx

```diff
@@ -1,13 +1,82 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
 import { TeamGrid } from './GridTeam'
-import { Story, Meta } from '@storybook/react'
 
-const Template: Story = (args) => <TeamGrid {...args} />
+export default {
+  title: 'Team Grid',
+  component: TeamGrid,
+} as ComponentMeta<typeof TeamGrid>
 
-export const Primary = Template.bind({})
+export const NoMembers: ComponentStory<typeof TeamGrid> = () => (
+  <TeamGrid team={fixtures.noMembers} />
+)
 
-Primary.args = {}
+export const WithMembers: ComponentStory<typeof TeamGrid> = () => (
+  <TeamGrid team={fixtures.withMembers} />
```

Comment:
This is a bit confusing. I would prop spread the whole thing if you can so you're spreading `withMembers`. Same with the other component

Reply:
sorry, I am not sure I understand - but I can see the grid and the images in storybook 🤔

Reply:
```js
<TeamGrid {...fixtures.withMembers} />

const fixtures = {
  withMembers: {
    team: [/* data */],
  }
}
```

Reply:
what does this serve though? isn't it the same?

Reply:
it's clearer & easier to add more props. Also `fixtures.withMembers` is a weird name to use for the team prop:

```jsx
<TeamGrid team={fixtures.withMembers} />
// vs
  <TeamGrid team={fixtures.teamMembers} />
```

But prop spreading is what I'd prefer to do for these stories, this only works because you are passing one prop at the moment.

Reply:
this to work though, it needs the `.team` at the end, it would complain before...
```jsx
  <TeamGrid team={...fixtures.teamMembers.team} />
```

Reply:
No it definitely works:

```js
import { ComponentMeta, ComponentStory } from '@storybook/react'

import { TeamGrid } from './GridTeam'

export default {
  title: 'Team Grid',
  component: TeamGrid,
} as ComponentMeta<typeof TeamGrid>

export const NoMembers: ComponentStory<typeof TeamGrid> = () => (
  <TeamGrid {...fixtures.noMembers} />
)

export const WithMembers: ComponentStory<typeof TeamGrid> = () => (
  <TeamGrid {...fixtures.withMembers} />
)

const fixtures = {
  noMembers: {
    team: [],
  },
  withMembers: {
    team: [
      {
        _key: '303774d9-252e-4626-90a3-633bceb1f736',
        image: {
          _type: 'image' as const,
          altText: null,
          asset: {
            _ref: 'image-5b9b97822d8d1ba9a6b6470596b0faa3e51c3797-1080x1080-png',
            _type: 'reference' as const,
          },
          dimensions: {
            _type: 'sanity.imageDimensions' as const,
            aspectRatio: 1,
            height: 1080,
            width: 1080,
          },
          mimeType: 'image/png',
        },
        job: 'Founder & Creative Director',
        name: 'Myles Palmer',
      },
      {
        _key: 'a4943e96-6397-4dfd-8037-37ea5ae517b9',
        image: {
          _type: 'image' as const,
          altText: null,
          asset: {
            _ref: 'image-97a970363f316fe8172858eb97517862ecc35a30-1616x1077-jpg',
            _type: 'reference' as const,
          },
          dimensions: {
            _type: 'sanity.imageDimensions' as const,
            aspectRatio: 1.500464252553389,
            height: 1077,
            width: 1616,
          },
          mimeType: 'image/jpeg',
        },
        job: 'Senior Digital Designer',
        name: 'Willem Purdy',
      },
      {
        _key: 'f2702b4a-18f1-4720-9c84-d404d3547383',
        image: {
          _type: 'image' as const,
          altText: null,
          asset: {
            _ref: 'image-f9a47493d6f82d32bcc5639b4cfad4043316169b-3024x1701-jpg',
            _type: 'reference' as const,
          },
          dimensions: {
            _type: 'sanity.imageDimensions' as const,
            aspectRatio: 1.7777777777777777,
            height: 1701,
            width: 3024,
          },
          mimeType: 'image/jpeg',
        },
        job: 'Fullstack Developer',
        name: 'Josh Ellis',
      },
    ],
  },
}
```